src/aks-preview/HISTORY.rst

```diff
@@ -12,6 +12,10 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 
+0.5.175
```

Comment:
Check with @FumingZhang  if you need to add a new version or add under 'Pending' section.

Reply:
I had added a new version number he gave us the option to do either